repository/q.json

```diff
@@ -444,6 +444,19 @@
 				}
 			]
 		},
+		{
+			"name": "Quote With Marker",
+			"author": "Denny Korsukéwitz",
+			"description": "This package quotes the selected area and adds a custom marker to it.",
```

Comment:
It's easier if you just put a description on your GitHub repo. Package control will know to find and use that.

Reply:
Ah, thanks for that hint.
Fixed with: https://github.com/wbond/package_control_channel/pull/8846/commits/c8946b2eeebe69494f09f57f191eeeff9c561b84